internal/discover/mounts_test.go

```diff
@@ -100,7 +100,7 @@ func TestMounts(t *testing.T) {
 				lookup: &lookup.LocatorMock{
 					LocateFunc: func(s string) ([]string, error) {
 						if s == "error" {
-							return nil, fmt.Errorf(s)
+							return nil, fmt.Errorf("error")
```

Comment:
nit: (Optional) If we are using constant strings, we can use the stdlib `errors.New` instead. Also we need a better error message than just `error` as it doesn't say much. I understand that it's a test case, but expressive error messages also help us better debug and trace unit test failures
```suggestion
							return nil, fmt.Errorf("error")
```

Reply:
Those are all valid points. Since this is a cherry-pick, I'm not going to address these here and would prefer to keep the functionality the same.